urdf2webots/parserURDF.py

```diff
@@ -421,53 +421,75 @@ def getSTLMesh(filename, node):
     return node
 
 
-def getOBJMesh(filename, node):
+def getOBJMesh(filename, node, link):
     """read obj file."""
-    print('Parsing Mesh: ' + filename)
+    print('Parsing Mesh: ' + filename)    
+    if hasattr(node, 'material') and node.material:
+        isVisual = True
+    else:
+        isVisual = False
+
     with open(filename, 'r') as file:
-        trimesh = node.geometry.trimesh
+        counter = 0
+        indexOffset = 0
         for line in file:
-            header, body = line.split(' ', 1)
+            header, body = line.split(' ', 1)            
```

Comment:
```suggestion
            header, body = line.split(' ', 1)
```

This kind of trailing spaces should be removed (it is by the way one of the reason of the test failure), I am sure you can configure your editor to remove them automatically when saving.

Reply:
I did a pep8 check, did not pick it up... hmmm